server/lib/tuist_web/router.ex

```diff
@@ -369,12 +369,11 @@ defmodule TuistWeb.Router do
       end
 
       scope "/cas" do
+        get "/prefix", CASController, :prefix
```

Comment:
would probably lift `prefix` to `/cache` since the `prefix` is used both for cas and kv